src/ApiHandler.ts

```diff
@@ -242,6 +245,7 @@ export default class ApiHandler {
 			onInitialize,
 			extrinsics,
 			onFinalize,
+			authorId,
```

Comment:
A nit, but this makes more sense to me by the header info. Right before or right after `logs`.